Car_Localizer/Car_Localizer/Base_loss_model/loss_model_base.py

```diff
@@ -3,23 +3,19 @@
 import torch.nn.functional as F
 
 from abc import ABCMeta, abstractmethod
-from Config.config import TensorType, ShapeType
+from Config.config import TensorType
 
 
 class Loss_Base(torch.nn.Module, metaclass=ABCMeta):
     def __init__(
         self,
-        name: str = "Loss_Base",
-        input_shape: ShapeType = (1, 3, 64, 64),
-        num_classes: int = 2,
+        name: str = "Loss_MSE",
```

Comment:
Loss_Base != Loss_MSE

just create another class, derive from Loss_Base and call it Loss_MSE